src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -942,10 +942,14 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
     <comment>{0} - Exception mssage</comment>
   </data>
   <data name="WhyCommand_Message_NonSDKStyleProjectsAreNotSupported" xml:space="preserve">
-    <value>Unable to run 'dotnet nuget why' for project '{0}'. This command only works on SDK-style projects.</value>
+    <value>Unable to run 'dotnet nuget why' for project '{0}'. See https://aka.ms/dotnet/nuget/why#older-project-format</value>
```

Comment:
```suggestion
     <value>Unable to run 'dotnet nuget why' for project '{0}'. See https://aka.ms/dotnet-nuget-why#older-project-format</value>
```

https://aka.ms/dotnet/nuget/why returns 404 whereas https://aka.ms/dotnet-nuget-why successfully navigates to dotnet nuget why docs.

There is another occurrence of the same incorrect URL in the below entry.

Reply:
I talked to the dotnet cli team and asked them what their prefered naming convention was, and they prefer slashes.  I just deleted https://aka.ms/dotnet-nuget-why and created https://aka.ms/dotnet/nuget/why.